app/views/api/projects/dependencies.json.jb

```diff
@@ -0,0 +1,30 @@
+dependencies_fields = {
+  dependencies_for_version: @version.number,
+  dependencies: (@version.dependencies.includes(:project) || []).
+    map {|dependency| DependencySerializer.new(dependency) }.
+    as_json
```

Comment:
I don't _think_ we need the as_json here? But not exactly sure how it figures it out. Perhaps I've been deluding myself.